src/python/pants/backend/jvm/tasks/jvm_compile/zinc/zinc_compile.py

```diff
@@ -455,6 +451,10 @@ def _compile_hermetic(self, jvm_options, ctx, classes_dir, jar_file,
         ),
       ])
 
+    # If analysis file exists, then incremental compile is enabled.
+    relpath_to_analysis = fast_relpath(ctx.analysis_file, get_buildroot())
+    merged_scratch_inputs = self._compute_scratch_inputs(classes_dir, relpath_to_analysis, jar_file)
```

Comment:
Please rename `scratch` to `local_only` or something throughout this file; imo scratch isn't self-explanatory enough to use it as a name

Reply:
Adjusted. Thanks!